osu.Game/Overlays/Profile/Sections/Ranks/DrawableProfileScore.cs

```diff
@@ -246,6 +247,9 @@ private Drawable createDrawablePerformance()
                 };
             }
 
+            if (Score.Beatmap?.Status.GrantsPerformancePoints() == true)
+                return new UnprocessedPerformancePointsPlaceholder { Size = new Vector2(16), Colour = colourProvider.Highlight1 };
```

Comment:
Isn't this wrong? In the case a score doesn't grant pp due to ranked status, it shouldn't show as "currently processing".

Reply:
Well, it's already done that way? If the beatmap grants pp and `score.PP == null` then it should show the unprocessed placeholder.

On a second look, the method reads completely back-to-front (and has been on master), so I'll fix that.

Reply:
Okay I was confused by the method ordering. Should be fine if it's rearranged.